examples/common/chip-app-server/include/server.h

```diff
@@ -0,0 +1,15 @@
+#ifndef CHIP_APP_SERVER_SERVER_H
+#define CHIP_APP_SERVER_SERVER_H
+
+#include <inet/IPAddress.h>
+#include <inet/InetLayer.h>
+#include <system/SystemPacketBuffer.h>
+#include <transport/SecureSessionMgr.h>
+#include <transport/UDP.h>
+
+using DemoSessionManager = chip::SecureSessionMgr<chip::Transport::UDP>;
+
+void StartServer(DemoSessionManager * sessions);
+void InitDataModelHandler();
```

Comment:
Would be good to have some documentation for these.

And if they're always expected to be called together (are they?) maybe one should just call the other....

Reply:
Done, StartServer -> InitServer and InitDataModelHandler is called in that function.